Kudu.Core/Tracing/KuduEvent.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kudu.Core.Tracing
+{
+    class KuduEvent
+    {
+        public string siteName = string.Empty;
+        public string projectType = string.Empty;
+        public string result = string.Empty;
+        public string error = string.Empty;
+        public string deploymentDurationInMilliseconds = string.Empty;
+        public string siteMode = string.Empty;
+        public string scmType = string.Empty;
+        public string vsProjectId = string.Empty;
+        public string jobName = string.Empty;
+        public string scriptExtension = string.Empty;
+        public string jobType = string.Empty;
+        public string trigger = string.Empty;
+        public string method = string.Empty;
+        public string path = string.Empty;
+        public string Message = string.Empty;
+        public string exception = string.Empty;
+        public string route = string.Empty;
+        public string userAgent = string.Empty;
+        public string requestId = string.Empty;
+        public string buildVersion = string.Empty;
+        public string address = string.Empty;
+        public string verb = string.Empty;
+        public int statusCode = 0;
+        public long latencyInMilliseconds = 0;
+
+        public override string ToString()
+        {
+            return $"{siteName},{projectType},{result},{NormalizeString(error)},{deploymentDurationInMilliseconds},{siteMode},{scmType},{vsProjectId}," +
```

Comment:
Alternatively, we could use `String.Join` here.

Reply:
I had explored that possibility since it might look cleaner. Couldn't personally find a way to join the fields of the class in a clean way. Would be possible to create an array of strings (casting any non-string fields to strings) and then join that. Not sure that is worth the hassle/if it would be any cleaner than this.